worker/src/RTC/SctpAssociation.cpp

```diff
@@ -356,15 +356,6 @@ namespace RTC
 
 		// Add maxMessageSize.
 		jsonObject["maxMessageSize"] = this->maxSctpMessageSize;
-
-		// Add sendBufferSize.
-		jsonObject["sendBufferSize"] = this->sctpSendBufferSize;
-
-		// Add sctpBufferedAmountLowThreshold.
-		jsonObject["sctpBufferedAmount"] = this->sctpBufferedAmount;
-
-		// Add isDataChannel.
-		jsonObject["isDataChannel"] = this->isDataChannel;
```

Comment:
Same. Can we please keep those? We don't need to remove them from `dump()` method whose output is not even documented (private API just for debugging). We really need them for debugging.

Reply:
I'm gonna do it

Reply:
done here: https://github.com/versatica/mediasoup/pull/518/commits/b2047f73512197086ec1129098c4be8e6fdf8e2b